controllers/suite_test.go

```diff
@@ -101,9 +108,56 @@ var _ = BeforeSuite(func() {
 		Expect(err).ToNot(HaveOccurred())
 	}()
 
+	// Create a shared configuration struct to pass Config information to all sub processes
+	sharedCfg := SharedConfig{
+		Host: cfg.Host,
+		TLSClientConfig: struct {
+			Insecure bool    `json:"insecure"`
+			CertData []uint8 `json:"certData,omitempty"`
+			KeyData  []uint8 `json:"keyData,omitempty"`
+			CAData   []uint8 `json:"caData,omitempty"`
```

Comment:
Could we define this as a type and avoid the repetition?

Reply:
Sounds good, done 👍